NEWS

```diff
@@ -74,6 +74,8 @@ Noteworthy changes in release 6.10 (2024-07-21)
   * Updated lists of BPF_*, IORING_*, KEXEC_*, KEY_*, LANDLOCK_*, PR_*, STATX_*,
     TCP_*, TEE_*, V4L2_*, and *_MAGIC constants.
   * Updated lists of ioctl commands from Linux 6.10.
+  * Added -e namespace=new option for printing the namespaces that the tracee
+    enters.
```

Comment:
This is being added to a historic (v6.10) news.

Reply:
fixed locally.